website/docs/plugins/optuna_sweeper.md

```diff
@@ -45,7 +44,7 @@ The default configuration is <GithubLink to="plugins/hydra_optuna_sweeper/hydra_
 
 ## Example 1: Single-Objective Optimization
 
-We include an example in this directory. `example/sphere.py` implements a simple benchmark function to be minimized.
+We include an example in [this directory](https://github.com/facebookresearch/hydra/tree/main/plugins/hydra_optuna_sweeper/example). `example/sphere.py` implements a simple benchmark function to be minimized.
```

Comment:
Thanks for adding this. could you update the link in both versions to use GithubLink? that way the link will be directed to the correct branch. Thank you.